src/plasmapy/plasma/grids.py

```diff
@@ -391,7 +391,7 @@ def unit(self):
     # *************************************************************************
     # 1D axes and step sizes (valid only for uniform grids)
     # *************************************************************************
-    @property
+    @cached_property
```

Comment:
All of these properties cannot be changed through the user accessible interface after initialization, so there's no reason to not cache them. The axes get called a fair bit in particle tracker, so it's nice to have them cached.